tests/integration/test_http_post.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import utils
+
+
+def test_http_post(test_env):
+    """
+    Send new request to cachito API
+
+    Checks:
+    * Check that response code is 201
+    * Check that response contains id number, same ref and repo as in request,
+        state_reason is: The request was initiated
+    """
+    client = utils.Client(test_env["api_url"])
+    response_created_req = client.create_new_request(
+        repo=test_env["post"]["repo"],
+        ref=test_env["post"]["ref"],
+        pkg_managers=test_env["post"]["pkg_managers"])
+
+    assert response_created_req.status == 201
+
+    assert "id" in response_created_req.data.keys()
```

Comment:
It would be great to also validate that in the `response_created_req`, `ref`, `repo`, and `pkg_managers` are correctly set.

We should also verify the `id` is an integer greater than 0.